bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TitleImpl.java

```diff
@@ -69,13 +87,25 @@ private void initModel() {
         if (StringUtils.isBlank(title)) {
             title = StringUtils.defaultIfEmpty(currentPage.getPageTitle(), currentPage.getTitle());
         }
-
         if (heading == null) {
             heading = Utils.Heading.getHeading(type);
             if (heading == null) {
                 heading = Utils.Heading.getHeading(currentStyle.get(PN_DESIGN_DEFAULT_TYPE, String.class));
             }
         }
+
+        if (StringUtils.isNotEmpty(linkURL)) {
+            linkURL = Utils.getURL(request, pageManager, linkURL);
+        } else {
+            linkURL = null;
+        }
+
+        if(currentStyle != null) {
+            linkDisabledObj = currentStyle.get(Title.PN_TITLE_LINK_DISABLED, Boolean.class);
```

Comment:
@Shivanshmathur - 
* Why both `linkDisabledObj` and `linkDisabled`? one variable `linkDisabled` should be enough
* We can just set an expected default (false) rather than using Boolean.class
* Can we use a consistent code style across the file, i.e. before the curly braces?

Reply:
Fixed.